plugins/snowflake/dbt/include/snowflake/macros/materializations/incremental.sql

```diff
@@ -0,0 +1,75 @@
+
+{% materialization incremental, adapter='snowflake' -%}
+
+  {%- set unique_key = config.get('unique_key') -%}
+  {%- set sql_where = config.get('sql_where') -%}
+  {%- set non_destructive_mode = (flags.NON_DESTRUCTIVE == True) -%}
+  {%- set full_refresh_mode = (flags.FULL_REFRESH == True) -%}
+  {%- set identifier = model['alias'] -%}
+
+  {%- set old_relation = adapter.get_relation(database=database, schema=schema, identifier=identifier) -%}
+  {%- set target_relation = api.Relation.create(database=database, identifier=identifier, schema=schema, type='table') -%}
+  {%- set exists_as_table = (old_relation is not none and old_relation.is_table) -%}
+  {%- set exists_not_as_table = (old_relation is not none and not old_relation.is_table) -%}
+  {%- set force_create = full_refresh_mode -%}
+
+  -- setup
+
+  {% set source_sql -%}
+     -- wrap sql in parens to make it a subquery --
```

Comment:
not a big deal, but we could remove the select here and just do:
```
{% set source_sql %}
(
  {{ sql }}
)
{% endset %}
```

because `sql` will always contain a `select` statement. This would just help make the compiled SQL that shows up in the logs and `target/` dir a little easier to parse but would be functionally equivalent